### ISSUE TITLE
[Docs] Create a shorter version of Business Edition installs for customer.appsmith.com

### ISSUE THREAD

**Issue by theparthacus** [appsmithorg/appsmith-docs#641]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Discord/slack/intercom Link

https://theappsmith.slack.com/archives/C02K2MZERSL/p1669964814284309

### Describe the problem

Our Business Edition installation guides today live contextually under the `Business Edition` on several docs.appsmith.com pages. They are too long to be shown in-app on customer.appsmith.com, our portal for Business users.

### Describe the improvement

To embed them on customer.appsmith.com, we need to make the content under those tabs much shorter and crisper. The preferable limit is five or so lines.

**Comment by pranayagarwal96** [comment]
Instructions for docs to install BE (both on hidden pages and main docs pages) to be updated as follows:

1. Step 1 (referring to this [doc](https://docs.appsmith.com/getting-started/setup/installation-guides/docker) stays the same, just we replace docker-compose.yml file with `ee` image instead of `ce` image; hence, Step 2 of that doc will no longer be required
2. Post Step 1, user will have to follow step 4 in this [doc](https://docs.appsmith.com/getting-started/setup/installation-guides/docker) (i.e. run the `docker-compose up -d` command)
3. Post this instance should be up and running, and once user creates an account, they will have to enter license key obtained from the Appsmith [customer portal](customer.appsmith.com) 

Once customer enters a valid license key, they should be good to get started on Appsmith

**Comment by jnikhila** [comment]
@pranayagarwal96 - From the existing Docker doc - as per your comment above - we need only three steps:

1. Click to download the [docker-compose.yml](Create_a_new_Bitly_Link) file and place it into the Appsmith installation folder.

OR, run the following cURL:

```
curl -L <ADD_NEW_BITYL_LINK_TO_DOWNLOAD -o $PWD/docker-compose.yml
```
2. Add the license key to the docker-compose.yml file as follows:
```
services:
   appsmith:
      # ...
      environment:
         APPSMITH_LICENSE_KEY: <YOUR_LICENSE_KEY>
```
3. Start the Docker container by running the command below. This may need to be run with sudo if docker and docker-compose aren't accessible by the user. If the image doesn't exist locally, this command downloads the necessary Docker image and starts the container.

```
docker-compose up -d
```
The Appsmith server should be up and running soon. Once the Appsmith container is ready, the message Appsmith is Running! will be displayed in the Docker logs. You can follow the logs with the following command:

```
docker logs -f appsmith
```
This message is also logged and available in server logs in (stacks/logs/backend/backend.log).

@sharat87 - Please verify and add if anything is missed.

**Comment by jnikhila** [comment]
@sharat87 - Could you please add pointers for Kubernetes doc?

**Comment by jnikhila** [comment]
@pranayagarwal96 - This ticket only caters to 2 pages for a shorter version of docker and Kubernetes installations. But, as per our earlier discussion, we had to create 4 pages - 2 for installs and 2 for an upgrade. Please confirm the understanding and let me know if the upgrades will be a part of a separate ticket.

**Comment by pranayagarwal96** [comment]
Yes, we will need 2 pages for upgrade as well. Let me know if I should create a separate ticket for upgrade pages or we can include as part of this ticket itself

**Comment by pranayagarwal96** [comment]
Don't think the existing pages will work. Just laying the content below for both Docker & Kubernetes (@sharat87 please check if this makes sense, especially for Kubernetes)

**For docker-compose:**
1. Step 1: Open the `docker-compose.yml` file is located. Update the image in `docker-compose.yml` file from `appsmith-ce` to `appsmith-ee`
2. Step 2: Recreate the instance by running the following command: `docker-compose up -d`
3. Step 3: Sign-in to your instance again. Upon successful login, a screen will be shown where you can enter your license key and activate the instance

**For Kubernetes**

1. Uninstall the community edition helm chart by running the following command: `helm uninstall appsmith`
2. Install the business edition helm chart by running the bfollowing commands:
```
helm repo add appsmith-ee https://helm-ee.appsmith.com
helm repo update
helm install appsmith appsmith-ee/appsmith-ee -n <namespace> -f values.yaml
```
3.  Sign-in to your instance again. Upon successful login, a screen will be shown where you can enter your license key and activate the instance

**Comment by Pranay105** [comment]
Here are the final links to all the hidden pages -
1. Install Docker BE - https://docs.appsmith.com/getting-started/setup/installation-guides/docker/install-business-edition
2. Upgrade docker CE to BE - https://docs.appsmith.com/getting-started/setup/upgrade-to-business-edition/upgrade-docker-to-business-edition
3. Install kubernetes BE - https://docs.appsmith.com/getting-started/setup/installation-guides/kubernetes/install-k8s-business-edition
4. Update kubernetes CE to BE - https://docs.appsmith.com/getting-started/setup/upgrade-to-business-edition/upgrade-k8s-to-business-edition
